ocaml/xapi-storage-script/main.ml

```diff
@@ -762,7 +768,20 @@ let vdi_of_volume x =
   ; cbt_enabled= Option.value x.Xapi_storage.Control.cbt_enabled ~default:false
   ; virtual_size= x.Xapi_storage.Control.virtual_size
   ; physical_utilisation= x.Xapi_storage.Control.physical_utilisation
-  ; sm_config= []
+  ; sm_config=
+      List.filter_map
+        (fun (k, v) ->
+          if String.starts_with ~prefix:_sm_config_prefix_key k then
```

Comment:
This would benefit from an comment that includes an example what is being transformed here and a motivation.